psse/psse-converter/src/main/java/com/powsybl/psse/converter/AreaConverter.java

```diff
@@ -0,0 +1,106 @@
+package com.powsybl.psse.converter;
```

Comment:
Can you add the copyright/licence header?

```java
/**
 * Copyright (c) 2025, <copyright>
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 */
```

On the first line, in replacement of `<copyright>`, you should use your institute/organization/university if you are contributing on its behalf, or your own name if it's on your own behalf.